packages/theming/theming-utils/package.json

```diff
@@ -30,16 +30,29 @@
   },
   "devDependencies": {
     "@fluentui-react-native/eslint-config-rules": "^0.1.1",
-    "@types/react-native": "^0.66.0",
     "@fluentui-react-native/scripts": "^0.1.1",
-    "react-native": "^0.66.0",
-    "react-native-windows": "^0.66.0"
+    "@types/react-native": "^0.66.0",
```

Comment:
Should also be 0.68?

Reply:
Yeah, I'm in the process of updating it across the repo. Seems that rnx-kit did not catch it.

Reply:
Types are updated to 0.68. I'll need to check if rnx-kit has a capability to update that. Still learning how it works.